pkg/cluster/sync.go

```diff
@@ -519,3 +530,52 @@ func (c *Cluster) syncDatabases() error {
 
 	return nil
 }
+
+func (c *Cluster) syncLogicalBackupJob() error {
+	var (
+		job        *batchv1beta1.CronJob
+		desiredJob *batchv1beta1.CronJob
+		err        error
+	)
+	c.setProcessName("syncing the logical backup job")
+
+	// sync the job if it exists
+	// NB operator pod at startup syncs all clusters, c.logicalBackupJob will be nil during such Sync
```

Comment:
Probably there is a lack of explanation why it would be nil?

Reply:
that looks like a dead comment ; I removed this pointer earlier